core/domain/exp_jobs_one_off_test.py

```diff
@@ -1355,6 +2175,187 @@ def test_for_migration_job(self):
+
+        exp_services.delete_exploration(self.albert_id, self.VALID_EXP_ID)
+
+        # Start ExplorationMigrationValidationJobForCKEditor.
+        job_id = (
+            exp_jobs_one_off.ExplorationMigrationValidationJobForCKEditor.create_new()) # pylint: disable=line-too-long
+        exp_jobs_one_off.ExplorationMigrationValidationJobForCKEditor.enqueue(
+            job_id)
+        self.process_and_flush_pending_tasks()
+
+        self.assertEqual(
+            exp_jobs_one_off.ExplorationMigrationValidationJobForCKEditor.get_output( # pylint: disable=line-too-long
+                job_id),
+            []
+        )
+
+
+class VerifyAllUrlsMatchGcsIdRegexJobTests(test_utils.GenericTestBase):
+
+    ALBERT_EMAIL = 'albert@example.com'
+    ALBERT_NAME = 'albert'
+
+    VALID_EXP_ID = 'exp_id0'
+    NEW_EXP_ID = 'exp_id1'
+    EXP_TITLE = 'title'
+
+    def setUp(self):
+        super(VerifyAllUrlsMatchGcsIdRegexJobTests, self).setUp()
+
+        # Setup user who will own the test explorations.
+        self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+        self.process_and_flush_pending_tasks()
+
+    def test_verification_of_image_and_audio_urls(self):
+        """Checks that image and auido urls are verified correctly."""
+
+        with self.swap(constants, 'DEV_MODE', False):
+
+            exploration = exp_domain.Exploration.create_default_exploration(
+                self.VALID_EXP_ID, title='title', category='category')
+
+            fs = fs_domain.AbstractFileSystem(
+                fs_domain.GcsFileSystem(self.VALID_EXP_ID))
+
+            with open(os.path.join(feconf.TESTS_DATA_DIR, 'img.png')) as f:
+                raw_image = f.read()
+
+            with open(os.path.join(feconf.TESTS_DATA_DIR, 'cafe.mp3')) as f:
+                raw_audio = f.read()
+
+            fs.commit(
+                self.albert_id, 'image/abc.png', raw_image,
+                mimetype='image/png'
+            )
+            fs.commit(
+                self.albert_id, 'image/abc.xyz', raw_image,
+                mimetype='image/png'
+            )
+            fs.commit(
+                self.albert_id, 'image/xyz/abc.png', raw_image,
+                mimetype='image/png'
+            )
+            fs.commit(
+                self.albert_id, 'audio/abc.mp3', raw_audio,
+                mimetype='audio/mp3'
+            )
+            fs.commit(
+                self.albert_id, 'audio/abc.png', raw_audio,
+                mimetype='audio/mp3'
+            )
+            fs.commit(
+                self.albert_id, 'audio/image/abc.mp3', raw_audio,
+                mimetype='audio/mp3'
+            )
+
+            exp_services.save_new_exploration(self.albert_id, exploration)
+
+            # Start VerifyAllUrlsMatchGcsIdRegex job on sample exploration.
+            job_id = exp_jobs_one_off.VerifyAllUrlsMatchGcsIdRegexJob.create_new() # pylint: disable=line-too-long
```

Comment:
Please resolve the line-too-long as stated above.

Reply:
Done.